src/plugins/intel_gpu/src/graph/impls/ocl/roi_align.cpp

```diff
@@ -91,11 +104,42 @@ struct roi_align_impl : typed_primitive_impl_ocl<roi_align> {
-                                           std::make_tuple(data_types::f32, format::bfyx),
-                                       });
+    implementation_map<roi_align>::add(impl_types::ocl,
+                                       roi_align_impl::create,
+                                       {std::make_tuple(data_types::u8, format::bfyx),
+                                        std::make_tuple(data_types::u8, format::b_fs_yx_fsv16),
+                                        std::make_tuple(data_types::u8, format::b_fs_yx_fsv32),
+                                        std::make_tuple(data_types::u8, format::bs_fs_yx_bsv16_fsv16),
+                                        std::make_tuple(data_types::u8, format::bs_fs_yx_bsv32_fsv16),
+                                        std::make_tuple(data_types::u8, format::bs_fs_yx_bsv32_fsv32),
+
+                                        std::make_tuple(data_types::i8, format::bfyx),
+                                        std::make_tuple(data_types::i8, format::b_fs_yx_fsv16),
+                                        std::make_tuple(data_types::i8, format::b_fs_yx_fsv32),
+                                        std::make_tuple(data_types::i8, format::bs_fs_yx_bsv16_fsv16),
+                                        std::make_tuple(data_types::i8, format::bs_fs_yx_bsv32_fsv16),
+                                        std::make_tuple(data_types::i8, format::bs_fs_yx_bsv32_fsv32),
+
+                                        std::make_tuple(data_types::i32, format::bfyx),
+                                        std::make_tuple(data_types::i32, format::b_fs_yx_fsv16),
+                                        std::make_tuple(data_types::i32, format::b_fs_yx_fsv32),
+                                        std::make_tuple(data_types::i32, format::bs_fs_yx_bsv16_fsv16),
+                                        std::make_tuple(data_types::i32, format::bs_fs_yx_bsv32_fsv16),
+                                        std::make_tuple(data_types::i32, format::bs_fs_yx_bsv32_fsv32),
+
+                                        std::make_tuple(data_types::f16, format::bfyx),
+                                        std::make_tuple(data_types::f16, format::b_fs_yx_fsv16),
+                                        std::make_tuple(data_types::f16, format::b_fs_yx_fsv32),
+                                        std::make_tuple(data_types::f16, format::bs_fs_yx_bsv16_fsv16),
+                                        std::make_tuple(data_types::f16, format::bs_fs_yx_bsv32_fsv16),
+                                        std::make_tuple(data_types::f16, format::bs_fs_yx_bsv32_fsv32),
+
+                                        std::make_tuple(data_types::f32, format::bfyx),
+                                        std::make_tuple(data_types::f32, format::b_fs_yx_fsv16),
+                                        std::make_tuple(data_types::f16, format::b_fs_yx_fsv32),
```

Comment:
Is this line mistake?

Reply:
wow, good catch!)
we clearly need a new approach here